association-frontend/package.json

```diff
@@ -35,21 +35,15 @@
     "@graphql-codegen/typescript-operations": "^1.17.14",
     "@graphql-codegen/typescript-react-apollo": "^2.2.1",
     "@graphql-codegen/typescript-urql": "^2.0.3",
+    "@python-italia/eslint-config-pycon": "^1.0.4",
```

Comment:
nice

Reply:
we could one for all our js apps instead of just pycon maybe? :D

Reply:
I can name the library "@python-italia/eslint-config-pythonit"

Reply:
Created `@python-italia/eslint-config` as global eslint config for python italia projects

Now we just need

```json
{
  "extends": ["@python-italia"]
}
```

to use it